lib/x11utils.pm

```diff
@@ -68,7 +68,17 @@ all possible options should be handled within loop to get unlocked desktop
 =cut
 sub ensure_unlocked_desktop {
     my $counter = 10;
-    while ($counter--) {
+
+    # check if the screenbuffer was not updated and record a softfailure in that case
+    if check_screen 'screenlock' {
+        send_key 'shift';
+        if check_screen 'screenlock' {
+            record_soft_failure 'bsc#1168979 screenbuffer containing old data';
+        }
+      }
+
+
+      while ($counter--) {
```

Comment:
something is fishy with the indention here

Reply:
Tidy might be messing around again, will update my packages soonish